src/GovUk.Education.ExploreEducationStatistics.Admin/Security/AuthorizationHandlers/CancelSpecificFileImportAuthorizationHandler.cs

```diff
@@ -25,7 +25,9 @@ public CancelSpecificFileImportAuthorizationHandler(IImportStatusService importS
             CancelSpecificFileImportRequirement requirement,
             ReleaseFileImportInfo import)
         {
-            if (await _importStatusService.IsImportFinished(import.ReleaseId, import.DataFileName))
+            var status = await _importStatusService.GetImportStatus(import.ReleaseId, import.DataFileName);
```

Comment:
Not a big deal, but wondering why we're making this an authorization level concern. Would have thought it makes more sense just being something the `ImportService` handles when you cancel it.

Reply:
The way I see it it shouldn't be allowed to cancel a completed import, for example, so I'd want this permission code to be clear on that as a rule.  In the same way that we don't allow people to update details / upload new files etc on a published Release, that status check is baked into the permission checks.

Reply:
Ah okay, yep fair enough!